iam.tf

```diff
@@ -21,6 +21,23 @@ data "aws_iam_policy_document" "sns_feedback" {
   }
 }
 
+// See https://repost.aws/knowledge-center/monitor-sns-texts-cloudwatch for required permissions to deliver status logs
+data "aws_iam_policy_document" "sns_feedback_allow_delivery_status_logs" {
+  count = local.create_sns_feedback_role ? 1 : 0
+
+  statement {
+    effect = "Allow"
+    actions = [
+      "logs:CreateLogGroup",
```

Comment:
Terraform creates the log group, this one can be removed

Reply:
Where is the log group being created? I see a log group for Lambda logs, but that is different. I believe this permission is necessary for the `sns/:region/:account/:topic` and `sns/:region/:account/:topic/Failure` log groups that are auto-created by SNS (assuming permissions are wired up correctly)